bindsnet/environment/__init__.py

```diff
@@ -213,11 +212,11 @@ def step(self, a: int) -> Tuple[torch.Tensor, float, bool, Dict[Any, Any]]:
         :return: Observation, reward, done flag, and information dictionary.
         """
         # Call gym's environment step function.
-        self.obs, self.reward, done, info = self.env.step(a)
+        self.obs, self.reward, self.done, info = self.env.step(a)
+        print(self.obs)
```

Comment:
Can you remove this print statement?

Reply:
Bump

Reply:
I actually removed this line. I guess this diff is outdated?